_includes/manuals/1.19/3.6_upgrade.md

```diff
@@ -126,7 +127,7 @@ You should compile i18n strings and precompile assets now:
 ##### Step 3 (A) - Database migration and cleanup
 
 The database should be migrated already, but you can make sure by executing the
-migration script again, it should produce no errors or output:
+migration script again, it should produce no errors or output. If you used foreman-installer to setup your existing Foreman instance, you can skip this step and jump over to step 3(C):
```

Comment:
I think the reason we included this section is just to make sure any errors during db migration are obvious - even if it means the user has to try to repeat the migration process after the installer does it.

Reply:
The installer doesn't clear the cache and existing sessions so totally jumping over it isn't correct. Running migrations and seeding doesn't hurt so I wouldn't skip it.